css/style.css

```diff
@@ -309,3 +338,105 @@ input {
   color: #ebebff;
   margin-bottom: 40px;
 }
+
+@media (min-width: 768px) {
+  .findMe {
+    width: 768px;
+    justify-content: space-between;
+  }
+
+  .work {
+    margin: 1px;
+    padding-left: 10px;
+    padding-right: 10px;
+  }
+
+  .habilities {
+    width: 600px;
+    margin-left: auto;
+    margin-right: auto;
+    grid-template-columns: auto auto auto;
+    grid-template-rows: auto auto;
+    place-items: center;
+  }
+
+  .description {
+    width: 600px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .title {
+    background-image: url("../pictures/twocirclesheader.png");
+    background-position: center;
+  }
+
+  .link8 {
+    display: inline;
+  }
+
+  .work1,
+  .work2,
+  .work3 {
+    width: 968px;
+    margin-left: auto;
+    margin-right: auto;
+    display: grid;
+    grid-template-columns: auto auto;
+    margin-top: 50px;
+    flex-wrap: wrap;
+  }
+
+  .jobdesc {
+    width: 450px;
+    height: 400px;
+  }
+
+  .work1title {
+    text-align: left;
+    padding-top: 0;
+    margin-left: 20px;
+    margin-top: 20px;
+    margin-bottom: 0;
+  }
+
+  body {
+    background-image: url("../pictures/Ellipse20.svg"), url("../pictures/Ellipse21.svg");
+    background-position: right, left;
+    background-position-y: 2000px, 2550px;
+  }
```

Comment:
- [ ] If you meant to use a relative path here then you should have only one dot at the beginning of the path, not two, because adding two dots it means it should look in the previous folder, kindly fix that accordingly. 😁